New version: BoundaryValueDiffEqFIRK v1.3.0

JuliaRegistrator opened the pull request:
- Registering package: BoundaryValueDiffEqFIRK
- Repository: https://github.com/SciML/BoundaryValueDiffEq.jl
- Created by: @ChrisRackauckas
- Version: v1.3.0
- Commit: 470f2b31876efc435675ed99ea8ee69b2ea40c0c
- Reviewed by: @ChrisRackauckas
- Reference: https://github.com/SciML/BoundaryValueDiffEq.jl/issues/45#issuecomment-2585161841
- Description: Boundary value problem (BVP) solvers for scientific machine learning (SciML)
<!-- 6471cd5cd7b041631498b2032e40c4dd961bfe9f69e8310c332466fd532d1df4820c49f07d9a3bb926f21a8932acaed58c6561fc02bcbb8fa8d6ed79faeae3aaf34a14e8c83b2ad4ad15e02ecec05657903f6f3f312cafa9cdbb5a17dea5ce503bedfd5709dccc453c6fc74a46fc16c83200be4b01d47d853175cfad979a5d873640fcb20210dd39374b27d9449c45ff8dfb44ff960aa0e13f044a67f4e51b3da1f0e6bef555a4fd6f43ac2861fac782f2314716b1711e4c0f52bf5bb9baf65be89743130a628f3c138d3f6891f7435e -->

ChrisRackauckas commented:
@ErikQQY can you check what happened with this one?

ErikQQY commented:
Subpackages require the latest(1.3.0) BoundaryVauleDiffEqCore which has just been tagged, so it would be fine after a few minutes

ChrisRackauckas commented:
ahhh

JuliaRegistrator committed:
New version: BoundaryValueDiffEqFIRK v1.3.0

UUID: 85d9eb09-370e-4000-bb32-543851f73618
Repo: https://github.com/SciML/BoundaryValueDiffEq.jl.git
Tree: feee7d8530e65c0ac38cc81321348c0a92c66f91

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

ErikQQY commented:
This PR need manual merging
